src/components/queueSettings/AdmissionControlPanel.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  CustomInput,
+  Button,
+  Input,
+  Form,
+  FormGroup,
+  Label,
+  Col,
+} from 'reactstrap'
+import { useBoolean, useInput } from 'react-hanger'
+
+const AdmissionControlPanel = ({ queue, updateQueue }) => {
+  const enabledToggle = useBoolean(queue.admissionControlEnabled)
+  const admissionControlUrl = useInput(queue.admissionControlUrl || '')
+  const changed =
+    enabledToggle.value !== queue.admissionControlEnabled ||
+    admissionControlUrl.value !== queue.admissionControlUrl
+
+  const update = () => {
+    updateQueue({
+      admissionControlEnabled: enabledToggle.value,
+      admissionControlUrl: admissionControlUrl.value,
+    })
+  }
+
+  const onSubmit = e => {
+    e.preventDefault()
+  }
+
+  return (
+    <Card className="mb-3">
+      <CardHeader>
+        <CardTitle tag="h5" className="mb-0">
+          Admission control
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <p>
+          You can set a programmatic admission control policy for your queue.
+          This allows you to disallow questions based on topic, rate-limit
+          students on the queue, or even maintain a blacklist of queue users.
+          For more information, check out the{' '}
```

Comment:
Not sure how I feel about using `{' '}` instead of joining the two lines together.

Reply:
This is a fun artifact of Prettier - 99% of the time I like what it does, but then it does things like this to keep line length short. I'm going to defer to Prettier here.